src/data-sources/DataSourceList.tsx

```diff
@@ -124,6 +134,11 @@ const DataSourceList = () => {
 		);
 	};
 
+	const getServiceLabel = ( service: ( typeof SUPPORTED_SERVICES )[ number ] ) => {
```

Comment:
Any particular reason of having this as a function and not using `SUPPORTED_SERVICES_LABELS` directly?

Maybe we should move this function to some common util file 🤔

Reply:
> Any particular reason of having this as a function and

Did that to avoid the lint error, and I couldn't find a way to add the ignore comment inside the JSX.